packages/file-type-icons/fileTypeIconMap.json

```diff
@@ -510,4 +510,4 @@
   "zip": {
     "extensions": ["zip"]
   }
-}
\ No newline at end of file
+}
```

Comment:
This new line was added after fixing the conflict.

Reply:
@tharshada can you please resolve this?